lib/node/nodes/gravity.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var Node = require('../node');
+
+var TYPE = 'gravity';
+
+var PARAMS = {
+    source: Node.PARAM.NODE(Node.GEOMETRY.ANY), // "populated places"
+    target: Node.PARAM.NODE(Node.GEOMETRY.POINT), // "malls"
+    weightColunn: Node.PARAM.STRING(),
```

Comment:
As @rochoa suggested in a previous pull request, we should use `snake_case` for node params.